### DISCUSSION TITLE
Nginx Manager 及 Let's Encrypt 簡易實作 | Ron Web - 探索學習的無限可能，分享技術的無盡價值

### DISCUSSION THREAD

**Discussion by giscus** [pzps94038/RonWeb#187] [Announcements]
# Nginx Manager 及 Let's Encrypt 簡易實作 | Ron Web - 探索學習的無限可能，分享技術的無盡價值

最近在架設部落格的伺服器，想要使用HTTPS的安全性，但憑證費用不是一筆小花費，在研究反向代理的時候，發現Nginx Manager整合了Let's Encrypt 的免費憑證申請，這篇文章就來實作這些內容

https://pzps94038.github.io/RonWeb/blog/article/25

**Comment by ioveasdkre** [comment]
感謝 羅哥分享，非常實用，一起加油，邁向程式巔峰！！！
